### PR TITLE
Feature/remove bootstrap from error pages

### PR DESCRIPTION
**Pull request by ibonkonesa**
Remove boostrat from error pages. 

Applied a pattern with only 1 master template (error) and others overrides message and icon using Twig include feature

### CODE REVIEW

**Comment by ibonkonesa** [comment]
This is the one!

**Comment by eerison** [comment]
Perfect, then when you commit again you won't close your PR :)

**Comment by ibonkonesa** [comment]
I understood that

**Comment by eerison** [comment]
@ibonkonesa other tip

When you open a Pull request isn't done yet, you put this one as Draft and after you finish all stuff you change to open, this way I know that PR is able to review :)

**Comment by ibonkonesa** [comment]
Thank you very much, I'm learning a lot this hacktoberfest thank to people like you :)

**Comment by ibonkonesa** [comment]
Nice! It this pull request ok? I only need this one to get the hacktoberfest prize. Can you please acccept this one?

**Comment by eerison** [comment]
Hi @ibonkonesa 

This error Page is different of the screen shot that I sent you

**Comment by eerison** [comment]
Please could you check the example that I've mentioned on the issue #270

File: templates/bundles/TwigBundle/Exception/error.html.twig [hunk 1]
```diff
@@ -1,61 +1,31 @@
...
-                        <div class="col-6">
-                            <a href="{{ path('app_profile', {'slug': app.user.slug}) }}" class="btn btn-primary btn-block"><i class="fa fa-user-circle"></i> My profile </a>
-                        </div>
-                    {% endif %}
-                </div>
-
-            </div>
-        </div>
+    <div class="container mt-5 has-text-centered">
+        <h1 class="is-size-4">{{ message| default('Internal server error') }}</h1>
```
Comment (eerison) [inline templates/bundles/TwigBundle/Exception/error.html.twig:7]: @ibonkonesa I guess bulma there's not any class `is-size-4`

**Review by eerison — CHANGES_REQUESTED** [review]

File: templates/bundles/TwigBundle/Exception/error.html.twig (hunk 1 above)
Comment (ibonkonesa) [reply templates/bundles/TwigBundle/Exception/error.html.twig:7]: Bulma uses this class:

https://bulma.io/documentation/helpers/typography-helpers/

**Comment by ibonkonesa** [comment]
If I followed the design suggested in #270 the including pattern would be not impossible but hard to afford. I think this design is nice, but I could apply the suggested using blocks instead of includes. Unfortunately, I can't do it now because my son needs playing :) but I could do it next week. Could you please accept this PR in order to I could get the Hacktoberfest prize? I want to collaborate with this project in the future :)

**Comment by eerison** [comment]
@ibonkonesa I've added `hacktoberfest-accepted` label :)

**Comment by eerison** [comment]
@ibonkonesa what I  mean is missing is for example

`5"icon"0` , `4"icon"4` etc...

**Comment by eerison** [comment]
@ibonkonesa any update?

**Pull request closed without merging**